<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsDialog</class>
 <widget class="QDialog" name="PrefsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>_(&quot;Preferences&quot;)</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxInterface">
     <property name="title">
      <string>_(&quot;Interface&quot;)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>_(&quot;Dictionary path:&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditIntfDicPath">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>_(&quot;Enter the dictionary path (mandatory to start a game)&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonIntfDicBrowse">
          <property name="toolTip">
           <string>_(&quot;Open a browser window to choose the dictionary&quot;)</string>
          </property>
          <property name="text">
           <string>_(&quot;Browse...&quot;)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>_(&quot;Definitions site:&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDefSite"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxIntfShowPoints">
        <property name="toolTip">
         <string>_(&quot;Show/hide the tiles points on the board.&quot;)</string>
        </property>
        <property name="text">
         <string>_(&quot;Display tiles points&quot;)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxIntfAlignHistory">
        <property name="toolTip">
         <string>_(&quot;If checked, the game and player histories will display the rack and the corresponding solution on the same line&quot;)</string>
        </property>
        <property name="text">
         <string>_(&quot;Align the rack and the solution in history&quot;)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxIntfWarnReplayTurn">
        <property name="toolTip">
         <string>_(&quot;If checked, a confirmation will be requested before replaying the game from a given turn. If unchecked, the action is performed without confirmation.&quot;)</string>
        </property>
        <property name="text">
         <string>_(&quot;Warn before replaying a turn&quot;)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>_(&quot;Timer total duration (in seconds):&quot;)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>_(&quot;Timer alert (in seconds):&quot;)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>_(&quot;Default computer level:&quot;)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBoxDefaultLevel">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxTimerTotal">
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBoxTimerAlert">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxDuplicate">
     <property name="title">
      <string>_(&quot;Duplicate mode&quot;)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxDuplRefuseInvalid">
        <property name="toolTip">
         <string>_(&quot;If checked, playing an invalid or misplaced word will not be possible. If unchecked, you will get 0 point and lose your turn&quot;)</string>
        </property>
        <property name="text">
         <string>_(&quot;Refuse to play invalid moves&quot;)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>_(&quot;Min. players for a solo:&quot;)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>_(&quot;Solo value:&quot;)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxDuplSoloValue">
        <property name="toolTip">
         <string>_(&quot;Value of the solo bonus. Set it to 0 if you don't want solo bonus&quot;)</string>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxDuplSoloPlayers">
        <property name="toolTip">
         <string>_(&quot;Minimum number of players needed to take into account the solo bonus&quot;)</string>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>checkBoxDuplRefuseInvalid</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>spinBoxDuplSoloValue</zorder>
     <zorder>horizontalSpacer</zorder>
     <zorder>spinBoxDuplSoloPlayers</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxFreeGame">
     <property name="title">
      <string>_(&quot;Free game mode&quot;)</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxFreeRefuseInvalid">
        <property name="toolTip">
         <string>_(&quot;If checked, playing an invalid or misplaced word will not be possible. If unchecked, you will get 0 point and lose your turn&quot;)</string>
        </property>
        <property name="text">
         <string>_(&quot;Refuse to play invalid moves&quot;)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTraining">
     <property name="title">
      <string>_(&quot;Training mode&quot;)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxIntfLinkTraining7P1">
        <property name="toolTip">
         <string>_(&quot;If checked, any change to the rack in arbitration mode will update the 'Plus 1' tab of the 'Dictionary tools' window&quot;)</string>
        </property>
        <property name="text">
         <string>_(&quot;Copy rack to the 'Plus 1' dictionary tool&quot;)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>_(&quot;Search results limit:&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxTrainSearchLimit">
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>_(&quot;Arbitration mode&quot;)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBoxArbitAutoMaster">
        <property name="toolTip">
         <string>_(&quot;If checked, a Master move will be selected by default when searching the results. It is still possible to change the Master move afterwards.&quot;)</string>
        </property>
        <property name="text">
         <string>_(&quot;Choose a Master move automatically&quot;)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxArbitLink7P1">
        <property name="toolTip">
         <string>_(&quot;If checked, any change to the player rack in training mode will update the 'Plus 1' tab of the 'Dictionary tools' window&quot;)</string>
        </property>
        <property name="text">
         <string>_(&quot;Copy rack to the 'Plus 1' dictionary tool&quot;)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>_(&quot;Search results limit:&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxArbitSearchLimit">
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditIntfDicPath</tabstop>
  <tabstop>pushButtonIntfDicBrowse</tabstop>
  <tabstop>lineEditDefSite</tabstop>
  <tabstop>checkBoxIntfShowPoints</tabstop>
  <tabstop>checkBoxIntfAlignHistory</tabstop>
  <tabstop>checkBoxIntfWarnReplayTurn</tabstop>
  <tabstop>spinBoxDefaultLevel</tabstop>
  <tabstop>spinBoxTimerTotal</tabstop>
  <tabstop>spinBoxTimerAlert</tabstop>
  <tabstop>checkBoxDuplRefuseInvalid</tabstop>
  <tabstop>spinBoxDuplSoloPlayers</tabstop>
  <tabstop>spinBoxDuplSoloValue</tabstop>
  <tabstop>checkBoxFreeRefuseInvalid</tabstop>
  <tabstop>checkBoxIntfLinkTraining7P1</tabstop>
  <tabstop>spinBoxTrainSearchLimit</tabstop>
  <tabstop>checkBoxArbitAutoMaster</tabstop>
  <tabstop>checkBoxArbitLink7P1</tabstop>
  <tabstop>spinBoxArbitSearchLimit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrefsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrefsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
